frontends/web/src/components/dialog/dialog.tsx

```diff
@@ -30,6 +30,7 @@ interface Props {
     disabledClose?: boolean;
     children: React.ReactNode;
     open: boolean;
+    fullscreenOnMobile?: boolean;
```

Comment:
note to myself: in last commit we try with View component, so if that goes well this can be undone